waku/v2/README.md

```diff
@@ -134,3 +134,23 @@ You can change this to `wakunode2`, the Waku v2 node like this:
 make docker-image MAKE_TARGET=wakunode2
 docker run --rm -it statusteam/nim-waku:latest --help
 ```
+## Using Websocket and secure Websockets
+
+Websocket config needs to be enabled , by default the port is 8000 for websockets.
+
+
+```
+# Run unsecure websockets
```

Comment:
```suggestion
# Run unsecure Websockets (doesn't require a certificate)
```

Reply:
resolved in 765a24e1549b18153b2313c61e305eddaacd2350